### DISCUSSION TITLE
How to delete old data, before transferring data.

### DISCUSSION THREAD

**Discussion by WeltonJerry** [apache/seatunnel#4122] [Q&A]
I want to transferring data to a MySql database. Maybe there is mistake in the data, so I want to rerun the transferring job after having fixed the issue.  Before doing that, I should clear the old error data. 

So my question is how to clear the old error data in my transferring job.

**Comment by lightzhao** [comment]
Can't you add a where filter to the sql transform?

**Reply by WeltonJerry** [reply]
It can only filter the data from source, but cannot delete the bad data from the target.

**Comment by SingleCloud** [comment]
Have the same question? Has a solution been found for it yet？
